xdocs/changes.xml

```diff
@@ -70,6 +70,18 @@ Summary
 
 <ch_section>Incompatible changes</ch_section>
 <ul>
+	<li>This version introduces a major change in the way SampleEvent (containing SampleResult) are delivered to Sample listeners like View Results Tree, Summarizer etc.
```

Comment:
Could we limit the line length here to 60-80 chars? While it won't affect the generated docs it makes it much easier to read the raw files and any changes

Reply:
> This version introduces a major change in the way SampleEvent (containing SampleResult

The wording sounds like a design detail rather than an explanation to the end-user.
I think something behind the lines of `View Results Tree, Summarizer performance is increased by 10x, in case jmx files exceeds 100MiB`.

Reply:
@vlsi , you are right about design detail.
But about your formulation, is it an example or really relate to this ?

I thought about "View Results Tree, Summarizer... performance is increased by 68% for high throughput test (high number of VUs or high request rate)"
Ok for you ?

Reply:
> Could we limit the line length here to 60-80 chars? While it won't affect the generated docs it makes it much easier to read the raw files and any changes

Hello,
done.
Thanks for review

Reply:
> View Results Tree... performance is increased by 68% for high throughput test

It seems odd as the typical recommendation is to disable `View Results Tree` for every performance test.

Reply:
It's ResultCollector, which is the listener that write the CSV/XML if no listener.
But this is implementation detail, so how to formulate it ?